README.md

```diff
@@ -186,11 +186,11 @@ You can also individually run the `generate_pages` and `test_functional` yarn sc
 
 To generate the pages manually you can run the `generate_pages` scripts with the schema directory. Run it from the `tests/functional` directory as follows:
 
-`./generate_pages.py ../../data/en/ ./generated_pages -r "../../base_pages"`
+`./generate_pages.py ../../tests/schemas/en/ ./generated_pages -r "../../base_pages"`
```

Comment:
Is this not meant to be:
```suggestion
`./generate_pages.py ../../tests_schemas/en/ ./generated_pages -r "../../base_pages"`
```

Reply:
No, it's test_schemas, but close enough.